.github/workflows/e2e_tests.yml

```diff
@@ -32,11 +32,14 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: '1.17.x'
+      - id: auth
+        uses: google-github-actions/auth@v0.4.2
+        with:
+          credentials_json: ${{ secrets.GCP_CI_SERVICE_ACCOUNT }}
```

Comment:
Can we file an issue to replace this with token federation?

Reply:
yes, but that requires additional setup. I can add an issue